src/components/versionista-info.jsx

```diff
@@ -19,55 +19,70 @@ import {dateFormatter} from '../scripts/formatters';
  */
 export default class VersionistaInfo extends React.Component {
   render () {
-    let message;
     if (!this.props.from
      || !this.props.to
      || this.props.from.source_type !== 'versionista'
      || this.props.to.source_type !== 'versionista') {
       return null;
     }
-    else if (this.props.from === this.props.to) {
-      message = <span>There is only one version. No diff to display.</span>;
+
+    return (
+      <div className="versionista-info">
+        {this.getMessage() || this.renderDiffLink()}
+      </div>
+    );
+  }
+
+  getMessage () {
+    let message = null;
+    if (this.props.from === this.props.to
+      || this.props.to.uuid === this.props.from.uuid) {
```

Comment:
This feels redundant; I think you could just get rid of the first line of this condition.

Reply:
Oh yea, I added the second condition `this.props.to.uuid===this.props.from.uuid` because of the first part of this [comment](https://github.com/edgi-govdata-archiving/web-monitoring-ui/pull/139#discussion_r146180899). At the time, I took it to mean that the `uuid`s could match but `from` and `to` could be different, but that doesn't sound right. I forgot to ask, but could you clarify what was meant there?

Reply:
> At the time, I took it to mean that the uuids could match but from and to could be different, but that doesn't sound right. I forgot to ask, but could you clarify what was meant there?

They can; you understood that right! But I was aiming at a different issue with that recommendation—you weren’t checking whether the two versions were identical at all, only that both were present.

My point at the time was *more* about the fact that you could have both a `from` and a `to` value that represented the same version than about whether the objects were the same. I recommended checking the actual `uuid` properties because they don’t necessarily have to be *the same object* to represent the same version, as you understood. Checking the actual object being identical first is redundant if you already know that both are present (which we do, because that is handled at the beginning of the `render()` method and this method is private).

Reply:
Ok, that's *very* good to know! Thanks for explaining 👍